source/leuven/data_management_guidelines.rst

```diff
@@ -0,0 +1,189 @@
+- If you or somebody else needs to perform extra (postprocessing) steps on your data, but you need to create space for computation of other projects/experiments, you can (temporarily) store data on your safe storage as well.
+- If you are computing on multiple machines (e.g. expensive computation on Tier-1 and postprocessing on Tier-2), your safe storage can be a sort of inbetween storage, especially when there is some time between those steps.
+
+
+What shouldn't you move to your safe storage:
+
+- Code or compiled software: in most cases, a version control system is a much better way to store your source code. Compiled software almost never makes sense to store, as in most cases it is built for a specific node architecture and operating system. Just store your source code, together with your compilation scripts, in a version controlled environment (code repository).
+- Public databases. Unless there is a high cost to downloading them, or you need to perform expensive postprocessing steps on them. Another positive usecase would be that you are working with automated workflows that allow you to read directly from your safe storage (advanced).
+- Unstructured directories from team members. This last one might be a bit less obvious, but it happens all the time that users create certain directories or files just to test or debug things. In many cases, these directories are not removed afterwards, and when that member leaves the group, it is not clear what the directory contains. It can then be very tempting to move the whole thing to your safe storage with the idea that this can be cleaned up later. As this almost never happens, it is much better to just try to figure out what they are, while still on staging. Having good guidelines on how to structure your directory tree, and asking team members to create such directories with for example a ``tmp`` prefix (or just in their own directories), will avoid such scenarios.
+
+
+How should you transfer your data
+---------------------------------
+
+There are multiple platforms available to transfer your data:
+
+- Globus: a platform that allows transfers in with their :ref:`web interface <globus-web>`, or their :ref:`command line interface <cli>`. There are multiple :ref:`managed collections <globus-collections>` available, including collections for ManGO, Tier-1 Data, OneDrive, the KU Leuven large volume storages... In case you want to transfer to or from somewhere where there is no managed collection, you can still create a :ref:`local endpoint <globus-local-endpoints>`. You can find the full documentation on our :ref:`Globus pages <globus platform>`.
+- iCommands/Python iRODS client/ManGO Portal: tools that allows users to transfer data to and from an iRODS-managed platform (i.e. ManGO and Tier-1 Data). The full documentation can be found on :ref:`VSC docs <clients>`.
```

Comment:
Is ManGO portal relevant here? You cannot transfer data between Tier-1 Data and Tier-2 via the portal.